### PR TITLE
E-comm brief (issue #64)

### PR DESCRIPTION
**Pull request by lrbloomfield**

### CODE REVIEW

File: _projects/information-architecture/shopping.md [hunk 1]
```diff
@@ -3,3 +3,73 @@ title: E-commerce
...
+----
+
+- Make use of card sorting to help determine the logical organization of the products
+- Devise a plan of attack
+- Timebox your work
+- Get feedback early & often
+- Iterate
+- Perfect is the enemy of good
+- Document your work and add it to your group folder on the Google Drive
+- Build out your presentation deck and your report as you go
```
Comment (nevanscott) [inline _projects/information-architecture/shopping.md]: No report on this one

**Review by nevanscott — DISMISSED** [review]
Looking good to me!

Maybe we should look this over together one more time before we launch the project, but I think it's ready to publish.

**Comment by nevanscott** [comment]
@lrbloomfield general note:

- When you're creating a pull request, make sure to give it a descriptive name, that you mention which issue it closes (if it does), and that you add it to the appropriate project (if applicable)

File: _projects/information-architecture/shopping.md [hunk 2]
```diff
@@ -3,3 +3,73 @@ title: E-commerce
...
+
+Requirements
+------------
+
+- In-person visit to the local retail store in order to get a feel for the needs and behaviors of users
+- User flow for discovery of a product
+- User flow for the checkout and purchase process 
+- Clear hierarchy and organization of product content 
+- Evaluate your design against heuristics and CRAP principles
+- Digitized screens (at a minimum of mid-fidelity)
```
Comment (nevanscott) [inline _projects/information-architecture/shopping.md]: Click-through prototype (medium fidelity)

**Review by dzc1 — APPROVED** [review]

**Pull request merged**